index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>progress bar</title>
+  </head>
+  <body>
+    <div class="frem">
+      <div class="logo">
+        <img src="image.png" alt="image logo" />
+      </div>
+      <p class="title-p">Images</p>
+      <div class="size-file">
+        <p>257.4 GB</p>
+        <p>12,847 files</p>
+      </div>
+      <div class="loading"></div>
+    </div>
+    <div class="frem">
+      <div class="logo">
+        <img src="image.png" alt="image logo" />
+      </div>
+      <p class="title-p">Videos</p>
+      <div class="size-file">
+        <p>4 GB</p>
+        <p>125 files</p>
+      </div>
+      <div class="loading-1"></div>
+    </div>
+    <div class="frem">
+      <div class="logo">
+        <img src="image.png" alt="image logo" />
```

Comment:
What does `alt` explain about the image??? i.e `image logo` ??

Reply:
the image added here is the symbol of image in any digital gallery hence i have kept it as image logo.